rails/app/javascript/components/App.jsx

```diff
@@ -100,6 +100,20 @@ class App extends Component {
           filterMap[category] = Array.from(speakerSet).sort();
           break;
         }
+        case FILTER_CATEGORIES[3]: {
+          // second category: Topic
+          const topicSet = new Set(
+            this.props.stories
+              .map(story => {
+                return story.points.map(
```

Comment:
@julinvictus 
2 things
1) you need to add the JSON field of topic to 
terrastories/rails/app/views/home/_home.json.jbuilder , probably in a similar way to the json.extract! story on line 2.
2) when iterating through the stories, i think story.points would be the places on the map, i think for a story field you coould just directly grab that from story.topic, once you do add it to the home json.

Reply:
also, not necessary but a jest test could help drive some of that relationship